modules/admin-ui/src/main/java/org/opencastproject/adminui/usersettings/persistence/UserSettingDto.java

```diff
@@ -29,20 +29,25 @@
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.Index;
+import javax.persistence.Lob;
 import javax.persistence.NamedQueries;
 import javax.persistence.NamedQuery;
 import javax.persistence.Table;
 import javax.persistence.UniqueConstraint;
 
 /** Entity object for user settings. */
 @Entity(name = "UserSettings")
-@Table(name = "oc_user_settings", uniqueConstraints = { @UniqueConstraint(columnNames = { "id"}) })
+@Table(name = "oc_user_settings", indexes = {
```

Comment:
I didn't find a way to generate this table.

Reply:
That's because the code for this got removed from the admin interface. I don't think there is a way of accessing this apart maybe from the REST api. This probably should be removed…